site/pages/search-results-contextual-fr.hbs

```diff
@@ -0,0 +1,94 @@
+---
+{
+	"title": "Résultats de la recherche",
+	"language": "fr",
+	"altLangPrefix": "recherche",
+	"pageType": "recherche",
+	"pageclass": "page-type-search",
+	"breadcrumb": [
+		{ "title": "Accueil", "link": "https://www.canada.ca/fr.html" }
+	],
+	"dateModified": "2018-12-21",
+	"share": "false",
+	"deptfeature": false
+}
+---
+<!--Note: visual example with styles applied: https://gc-proto.github.io/student-loans/scenarioA/services/search-results-contextual-visual.html-->
+<!--Note 2: this template should only be used for contextual search results from a single institution-->
+<!--Note 3: until radio buttons pattern works, please simply include 'All government of Canada websites' link in the well, under the search box-->
+<h1 property="name">Résultats de la recherche</h1>
+<div class="well">
+	<form name="cse-search-box" method="GET" action="/en/sr/srb.html" id="cse-search-box">
+		<label class="wb-inv" for="sch-inp">Recherchez le site Web</label>
+		<!--search engine form submission code-->
+		<div class="input-group mrgn-tp-lg">
+			<input value="[value]" spellcheck="false" autocomplete="off" dir="ltr" id="sch-inp" name="q" class="form-control" type="search">
+			<span class="input-group-btn">
+				<button type="submit" class="btn btn-primary btn-small" name="wb-srch-sub"> <span class="glyphicon-search glyphicon"></span><span class="wb-inv">Recherche</span></button>
+			</span>
+		</div>
+	</form>
+	<p class="wb-inv">Voire les résultats de:</p>
+	<ul class="inline">
+		<li><input type="radio" name="searchscope" id="radio-allresults" value="allresults"  /> <label id="label-alldepartments">Tous les sites Web du gouvernement du Canada</label></li>
+		<li><input type="radio" name="searchscope" id="radio-context" value="context" checked="checked" /> <label id="label-singledepartment"><strong>[context label]</strong></label></li>
+	</ul>
```

Comment:
Having those input radio after the submit button could create an info and relationship/meaningful sequence issue. An alternative would be to have the submit button after those radio button. I know we have discussed to replace the input by a link which could technically "avoid" that issue but it may create some confusion as it will be still grouped together. I will draft some design alternative for further discussion, like using the tabbed interface or having the submit button after the input radio.